server/filesystem/archive.go

```diff
@@ -62,8 +62,21 @@ func (a *Archive) Create(dst string) error {
 		writer = f
 	}
 
+	// The default compression level is BestSpeed
+	var cl = pgzip.BestSpeed
+
+	// Choose which compression level to use based on the compression_level configuration option
+	switch config.Get().System.Backups.CompressionLevel {
+	case "none":
+		cl = pgzip.NoCompression
+	case "best_speed":
+		cl = pgzip.BestSpeed
+	case "best_compression":
+		cl = pgzip.BestCompression
+	}
+
 	// Create a new gzip writer around the file.
-	gw, _ := pgzip.NewWriterLevel(writer, pgzip.BestSpeed)
+	gw, _ := pgzip.NewWriterLevel(writer, cl)
```

Comment:
Why would you default to no compression? I get that it's what you want the majority of users probably aren't going to want that.

Reply:
I might be missing something here but cl should be BestSpeed unless it's manually changed.
I've tested this with a config file that doesn't have the "compression_level" set and the backups this created had a 22.7% compression ratio, so it did compress them. After starting my modified version of wings the config file did also have the "best_speed" option set. 

When I manually changed the compression_level to "none" the files were much larger and had a compression ratio of 0%.

Reply:
I misread the case statement.

I still think this is a very niche issue.

Reply:
Guess I can't argue with that. Although some people might also benefit from being able to select a higher compression level.